README.md

```diff
@@ -174,21 +175,6 @@ methods are often the simplest and easiest way to contribute.
 You can also simply run `./whats_left.sh` to assist in finding any unimplemented
 method.
 
-## Using a different standard library
-
-As of now the standard library is under construction. You can change a standard
-library by setting the RUSTPYTHONPATH environment variable.
-
-To do this, follow this method:
-
-```shell
-$ export RUSTPYTHONPATH=./Lib  # this is same as the default value
-$ cargo run -- -c 'import xdrlib'
-```
-
-You can play around with other standard libraries for python. For example, the
-[ouroboros library](https://github.com/pybee/ouroboros) or CPython Lib.
```

Comment:
I think this is still a good resource for development to add new standard library. Moving to development document?

Reply:
I don't know, I don't think there's much of a use case for this section anymore -- this has been here since before `__import__` with importlib was implemented, and all the modules we had were Rust native modules. Now, I think we're pretty settled on modified CPython Lib+some native modules, so this isn't very necessary for development anymore.